inst/rmarkdown/templates/pihph/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: This is a title and this is too
+subtitle: A subtitle goes on another line
+# Use the following to specify a shorter title for the header.
+title-header: A short title
+author:
+- name: Author's name
+  affiliation: Author's affiliation
+  email: e-mail@uni.edu
+- name: Second author's name (if there is one)
+  affiliation: Second author's affiliation
+# Use the following to specify a short author name for the header.
+author-header: Authors
+output: rticles::pihph_article
+gloss: expex # feel free to use your preferred package for the examples
+# FONTS
+# If default-fonts is true, alt-fonts should be set to false, and vice versa.
+default-fonts: false # set to true if you want to use the defaults fonts
+# The default fonts are: Cambria, Calibri, and Lekton
+# Lekton can be found at: https://www.fontsquirrel.com/fonts/lekton
+alt-fonts: true # set to true if you want to use alternative fonts
```

Comment:
In fact we have an issue in the CI because we are using `alt-fonts` with this example.

Do you have those fonts ? Is it easy to install on Linux or with Latex ? 

This should be well explained as this could definitely cause issue with users trying to use the default skeleton.

I believe they can be installed using CRAN so maybe we could do something to have them install by default 🤔

Reply:
I made a change in **tinytex** (https://github.com/yihui/tinytex/pull/268) that solves the issue for all the fonts except Lekton. 

This font is not available for latex in CTAN so it must be installed on the system. 

Maybe we can just default to xelatex font here and sets everything to FALSE ?

Reply:
I changed how fonts are handled in the template, and updated the documentation in the yaml preamble. Now there simply are parameters for `mainfont`, `licensefont`, and `logofont`. This allows for more flexibility and users can choose the typeface based on what they have on system.